src/RadioButton.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { color } from 'styled-system'
+import Icon from './Icon'
+
+const RadioButton = ({
+  className,
+  theme,
+  onClick,
+  isSelected,
+  disabled,
+  radioButtonText
+}) => {
+  const radioIconName = isSelected ? 'radioFilled' : 'radio'
+  const radioIconColor = isSelected ? 'blue' : 'borderGray'
+
+  function handleClick() {
+    if (disabled || isSelected) return
+    onClick()
+  }
+
+  return (
+    <RadioButtonWrap
+      className={className}
+      onClick={handleClick}
+      isSelected={isSelected}
+      disabled={disabled}
+      theme={theme}
+    >
+      <RadioIconWrap>
+        <Icon name={radioIconName} size={24} color={radioIconColor} />
+      </RadioIconWrap>
+      <RadioButtonText>{radioButtonText}</RadioButtonText>
```

Comment:
Nonblocking: you could consider designating `radioButtonText` as `children` to allow for

```
<RadioButton>Hello</RadioButton
<RadioButton>Bonjour</RadioButton
<RadioButton>Hola</RadioButton
```